cxx-sensors/src/main/resources/compiler-vc.xml

```diff
@@ -14171,6 +14174,81 @@ This warning indicates that a function is being used that has been banned, and h
     <remediationFunction>LINEAR</remediationFunction>
     <remediationFunctionGapMultiplier>5min</remediationFunctionGapMultiplier>
   </rule>
- 
+  
+<!-- ########### VS2017 15.7 ########### -->
```

Comment:
Please do not create a new section with version specific updates. The new rules should be added with internal key number ascending order.
We can always use Git history and commit comments to identify 'VS2017 15.7' enhancements.